# SOME DESCRIPTIVE TITLE.
# Copyright (C) YEAR ORGANIZATION
# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
#
msgid ""
msgstr ""
"Project-Id-Version: PACKAGE VERSION\n"
"POT-Creation-Date: 2007-06-03 15:45+Paris, Madrid\n"
"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
"Language-Team: LANGUAGE <LL@li.org>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=CHARSET\n"
"Content-Transfer-Encoding: ENCODING\n"
"Generated-By: pygettext.py 1.5\n"


#: gpicsync-GUI.py:124
msgid "Local time correction"
msgstr ""

#: gpicsync-GUI.py:126
msgid "Language"
msgstr ""

#: gpicsync-GUI.py:128
msgid "&Options"
msgstr ""

#: gpicsync-GUI.py:130
msgid "About..."
msgstr ""

#: gpicsync-GUI.py:132
msgid "&Tools"
msgstr ""

#: gpicsync-GUI.py:133
msgid "EXIF reader"
msgstr ""

#: gpicsync-GUI.py:134
msgid "Geo-Rename pictures"
msgstr ""

#: gpicsync-GUI.py:135 gpicsync-GUI.py:760
msgid "GPX Inspector"
msgstr ""

#: gpicsync-GUI.py:136
msgid "KMZ Generator"
msgstr ""

#: gpicsync-GUI.py:137
msgid "&Help"
msgstr ""

#: gpicsync-GUI.py:146
msgid "Pictures folder"
msgstr ""

#: gpicsync-GUI.py:147
msgid "GPS file (.gpx)"
msgstr ""

#: gpicsync-GUI.py:148
msgid " Synchronise ! "
msgstr ""

#: gpicsync-GUI.py:149 gpicsync-GUI.py:571
msgid "Quit"
msgstr ""

#: gpicsync-GUI.py:150
msgid "Stop"
msgstr ""

#: gpicsync-GUI.py:151
msgid "Clear"
msgstr ""

#: gpicsync-GUI.py:152
msgid "View in Google Earth"
msgstr ""

#: gpicsync-GUI.py:154
msgid "UTC Offset="
msgstr ""

#: gpicsync-GUI.py:155
msgid "Geocode picture only if time difference to nearest track point is below (seconds)="
msgstr ""

#: gpicsync-GUI.py:156
msgid "Create a log file in picture folder"
msgstr ""

#: gpicsync-GUI.py:158
msgid "Dates must match"
msgstr ""

#: gpicsync-GUI.py:160
msgid "Create a Google Earth file"
msgstr ""

#: gpicsync-GUI.py:162
msgid "with TimeStamp"
msgstr ""

#: gpicsync-GUI.py:164
msgid "Google Maps export, folder URL="
msgstr ""

#: gpicsync-GUI.py:168
msgid "backup pictures"
msgstr ""

#: gpicsync-GUI.py:170
msgid "interpolation"
msgstr ""

#: gpicsync-GUI.py:172
msgid "add geonames and geotagged"
msgstr ""

#: gpicsync-GUI.py:256
msgid "Choose a language"
msgstr ""

#: gpicsync-GUI.py:256
msgid "languages choice"
msgstr ""

#: gpicsync-GUI.py:317
msgid "More informations and help:"
msgstr ""

#: gpicsync-GUI.py:334
msgid "To visualize the results in Google Earth you must either:"
msgstr ""

#: gpicsync-GUI.py:335 gpicsync-GUI.py:753
msgid "- finish a synchronisation"
msgstr ""

#: gpicsync-GUI.py:347
msgid "Couldn't find or launch Google Earth"
msgstr ""

#: gpicsync-GUI.py:388
msgid "You must first select a pictures folder and a GPX file."
msgstr ""

#: gpicsync-GUI.py:402
msgid "Beginning synchronization with "
msgstr ""

#: gpicsync-GUI.py:403
msgid "UTC Offset ="
msgstr ""

#: gpicsync-GUI.py:404
msgid " hours and maximum time difference = "
msgstr ""

#: gpicsync-GUI.py:404
msgid " seconds"
msgstr ""

#: gpicsync-GUI.py:414
msgid "Creating an 'originals-backup' folder."
msgstr ""

#: gpicsync-GUI.py:421
msgid "Starting to generate a Google Earth file (doc.kml) in the picture folder ..."
msgstr ""

#: gpicsync-GUI.py:425
msgid "Starting to generate a Google Map file (doc-web.kml) in the picture folder"
msgstr ""

#: gpicsync-GUI.py:436
msgid "Geocoded with UTC Offset= "
msgstr ""

#: gpicsync-GUI.py:437
msgid " and  Maximum time difference = "
msgstr ""

#: gpicsync-GUI.py:439
msgid "Pictures Folder: "
msgstr ""

#: gpicsync-GUI.py:440
msgid "GPX file: "
msgstr ""

#: gpicsync-GUI.py:456
msgid "(Found "
msgstr ""

#: gpicsync-GUI.py:467
msgid "Processed image "
msgstr ""

#: gpicsync-GUI.py:491
msgid "Couldn't retrieve geonames data..."
msgstr ""

#: gpicsync-GUI.py:514
msgid " (writing geonames and geotagged to keywords tag in picture EXIF)"
msgstr ""

#: gpicsync-GUI.py:522
msgid "FINISHED GEOCODING PROCESS"
msgstr ""

#: gpicsync-GUI.py:524
msgid "PROCESSING STOPPED BY THE USER"
msgstr ""

#: gpicsync-GUI.py:528
msgid "Adding the GPS track log to the Google Earth kml file"
msgstr ""

#: gpicsync-GUI.py:531
msgid "Click on the 'View in Google Earth' button to visualize the result"
msgstr ""

#: gpicsync-GUI.py:532
msgid "( A Google Earth doc.kml file has been created in your picture folder.)"
msgstr ""

#: gpicsync-GUI.py:538
msgid "( A Google Maps doc-web.kml file has been created with the given url )"
msgstr ""

#: gpicsync-GUI.py:558
msgid "Local time corrections"
msgstr ""

#: gpicsync-GUI.py:561
msgid "Use this option ONLY if your camera local time is wrong."
msgstr ""

#: gpicsync-GUI.py:562
msgid "Indicate here the local time now displayed by your camera and GPS (hh:mm:ss)"
msgstr ""

#: gpicsync-GUI.py:564
msgid "Local time displayed now by the camera"
msgstr ""

#: gpicsync-GUI.py:567
msgid "Local time displayed now by the GPS"
msgstr ""

#: gpicsync-GUI.py:570
msgid "Apply correction"
msgstr ""

#: gpicsync-GUI.py:591
msgid "EXIF Reader"
msgstr ""

#: gpicsync-GUI.py:594
msgid "Read the EXIF metadata of the selected picture."
msgstr ""

#: gpicsync-GUI.py:596 gpicsync-GUI.py:597 gpicsync-GUI.py:628
msgid "All EXIF metadata"
msgstr ""

#: gpicsync-GUI.py:598 gpicsync-GUI.py:631
msgid "Date/Time/Lat./Long."
msgstr ""

#: gpicsync-GUI.py:604
msgid "Select a picture"
msgstr ""

#: gpicsync-GUI.py:626
msgid "Selected metada "
msgstr ""

#: gpicsync-GUI.py:636
msgid "lat./long."
msgstr ""

#: gpicsync-GUI.py:638
msgid "None"
msgstr ""

#: gpicsync-GUI.py:644
msgid "Renaming tool"
msgstr ""

#: gpicsync-GUI.py:647
msgid "This tool renames your pictures with the "
msgstr ""

#: gpicsync-GUI.py:647
msgid "original time/date and lat./long.(if present)"
msgstr ""

#: gpicsync-GUI.py:649
msgid "Rename pictures in a folder"
msgstr ""

#: gpicsync-GUI.py:650
msgid "Rename a single picture"
msgstr ""

#: gpicsync-GUI.py:668 gpicsync-GUI.py:689
msgid "Beginning renaming..."
msgstr ""

#: gpicsync-GUI.py:676 gpicsync-GUI.py:703
msgid "Didn't find Original Time/Date for "
msgstr ""

#: gpicsync-GUI.py:679 gpicsync-GUI.py:710
msgid "Renamed "
msgstr ""

#: gpicsync-GUI.py:693
msgid "Interrupted by the user"
msgstr ""

#: gpicsync-GUI.py:711
msgid "Finished"
msgstr ""

#: gpicsync-GUI.py:718
msgid "Create a kmz file archive"
msgstr ""

#: gpicsync-GUI.py:720
msgid "Create KMZ file !"
msgstr ""

#: gpicsync-GUI.py:728
msgid "Sorry this tool is not yet available for the Linux version"
msgstr ""

#: gpicsync-GUI.py:737
msgid "Creating a KMZ file in the pictures folder..."
msgstr ""

#: gpicsync-GUI.py:740
msgid "Adding doc.kml"
msgstr ""

#: gpicsync-GUI.py:744
msgid "Adding "
msgstr ""

#: gpicsync-GUI.py:748
msgid "KMZ file created in pictures folder"
msgstr ""

#: gpicsync-GUI.py:750
msgid "Couldn't rename the zip file to kmz (maybe a previous file already exist)"
msgstr ""

#: gpicsync-GUI.py:752
msgid "To create a Google Earth kmz file you must either:"
msgstr ""

#: gpicsync-GUI.py:754
msgid "- select a folder you've already synchronized then select the KMZ Generator tool"
msgstr ""

#: gpicsync-GUI.py:762
msgid "Inspect a gpx file and show tracklog data."
msgstr ""

#: gpicsync-GUI.py:764
msgid "Select a gpx file"
msgstr ""

#: gpicsync-GUI.py:783
msgid "Select a gpx file first."
msgstr ""

#: gpicsync-GUI.py:786
msgid "Looking at "
msgstr ""

#: gpicsync-GUI.py:787
msgid "Number of valid track points found"
msgstr ""

#: gpicsync-GUI.py:790
msgid "Date"
msgstr ""

#: gpicsync-GUI.py:790
msgid "Time"
msgstr ""

#: gpicsync-GUI.py:791
msgid "Latitude"
msgstr ""

#: gpicsync-GUI.py:792
msgid "Longitude"
msgstr ""

#: gpicsync-GUI.py:793
msgid "Altitude"
msgstr ""

#: gpicsync.py:268
msgid "taken "
msgstr ""

#: gpicsync.py:269 gpicsync.py:289
msgid "writing best latitude/longitude match to picture: "
msgstr ""

#: gpicsync.py:270 gpicsync.py:276 gpicsync.py:290 gpicsync.py:297
msgid " time difference (s)= "
msgstr ""

#: gpicsync.py:275
msgid "no track point below the maximum time range "
msgstr ""

#: gpicsync.py:275 gpicsync.py:279 gpicsync.py:297
msgid " : WARNING: DIDN'T GEOCODE, "
msgstr ""

#: gpicsync.py:277
msgid " long="
msgstr ""

#: gpicsync.py:277
msgid "For information nearest trackpoint was at lat="
msgstr ""

#: gpicsync.py:279
msgid "no track point at this picture date "
msgstr ""

#: gpicsync.py:292
msgid "Warning: Picture date "
msgstr ""

#: gpicsync.py:293
msgid " and track point date "
msgstr ""

#: gpicsync.py:293
msgid " are different ! "
msgstr ""

#: gpicsync.py:297
msgid " no suitable track point below maximum time difference, "
msgstr ""

